### PR TITLE
Introduce [decisions and rules]-embedded-mode-example

### PR DESCRIPTION
**Pull request by jomarko**
This PR is related to the https://github.com/IBM/bamoe-docs/pull/118

### CODE REVIEW

File: decisions-embedded-mode-example/src/main/java/com/ibm/bamoe/decisions/embedded/DecisionsEmbeddedModeExample.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        ExecutionResults executionResults = kieSession.execute(
+            CommandFactory.newBatchExecution(Arrays.asList(
+                CommandFactory.newInsert(new Applicant("#0001", 20), "applicant"),
+                CommandFactory.newInsert(new LoanApplication("#0001"), "application"),
+                new SetActiveAgendaGroup("applicationGroup"),
+                CommandFactory.newFireAllRules()
+            ))
+        );
+
+        System.out.println(executionResults.getResults().get("application"));
```
Comment (gitgabrio) [inline decisions-embedded-mode-example/src/main/java/com/ibm/bamoe/decisions/embedded/DecisionsEmbeddedModeExample.java]: Hi @jomarko 
Pls replace System.out.println with logger usage

File: decisions-embedded-mode-example/src/main/java/com/ibm/bamoe/decisions/embedded/DecisionsEmbeddedModeExample.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+        kieSession.dispose();
+
+        System.out.println("-----> Now we execute rules in the stateless session <-----");
```
Comment (gitgabrio) [inline decisions-embedded-mode-example/src/main/java/com/ibm/bamoe/decisions/embedded/DecisionsEmbeddedModeExample.java]: I'm confused, but IINW there is no relationship with DRL execution and DMN one, and I can't see any embedding: could you please clarify that ?

**Review by gitgabrio — CHANGES_REQUESTED** [review]
HI @jomarko 
Thanks for PR, but I have the impression we are mixing two different things here `decisions`(DMN) and `rules` (DRL)
The current documentation does not clearly separate them because, until now, we dealt only with DMN, but they should be clearly separated, IMO (even in the code example).

@baldimir @porcelli

File: rules-embedded-mode-example/src/main/java/com/ibm/bamoe/rules/embedded/RulesEmbeddedModeExample.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+                new SetActiveAgendaGroup("applicationGroup"),
+                CommandFactory.newFireAllRules()
+            ))
+        );
+
+        System.out.println(executionResults.getResults().get("application"));
+
+        kieSession.dispose();
+
+        System.out.println("-----> Now we execute rules in the stateless session <-----");
```
Comment (gitgabrio) [inline rules-embedded-mode-example/src/main/java/com/ibm/bamoe/rules/embedded/RulesEmbeddedModeExample.java]: Hi @jomarko 
Could you please replace System.out with proper logging ?  Thanks!

**Review by gitgabrio — CHANGES_REQUESTED** [review]
Thanks for the PR. There is just one thing to fix, i.e. pls replace all System.out with proper logging. Beside that, everything seems fine

**Comment by jomarko** [comment]
@gitgabrio comments addressed .

**Review by gitgabrio — APPROVED** [review]

**Pull request merged**